src/ofxGLWarper.cpp

```diff
@@ -315,8 +311,28 @@ void ofxGLWarper::setAllCorners(glm::vec2& top_left, glm::vec2 &top_right, glm::
     processMatrices();
 }
 //--------------------------------------------------------------
-glm::vec2 ofxGLWarper::getCorner(CornerLocation cornerLocation){
-    return corners[cornerLocation];// * glm::vec2(width, height);
+void ofxGLWarper::moveSurface(glm::vec2 &moveBy){
+    for (int i = 0; i < 4; ++i) {
+        corners[static_cast<CornerID>(i)] += moveBy;
```

Comment:
please remove the static_cast. I does not make sense. the arrays' index is an int. as such, having it cast to CornerID will have it cast automatically again back to int.

Reply:
Sure, I got confused